### PR TITLE
veusz: Add version 3.1

### PR DESCRIPTION
**Pull request by tokiedokie**
Created `veusz.json`
this is a plotting software for science

### CODE REVIEW

File: bucket/veusz.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        "github": "https://github.com/veusz/veusz"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/veusz/veusz/releases/download/veusz-$version/veusz-$version-windows-setup.exe"
+            }
+        }
+    }
+}
```
Comment (Ash258) [inline bucket/veusz.json]:
```suggestion
}

```

File: bucket/veusz.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+    "homepage": "https://veusz.github.io",
+    "description": "a scientific plotting package",
```
Comment (Ash258) [inline bucket/veusz.json]:
```suggestion
    "description": "Scientific plotting and graphing program, designed to produce publication-ready 2D and 3D plots.",
```

File: bucket/veusz.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+            "Veusz"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/veusz/veusz"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/veusz/veusz/releases/download/veusz-$version/veusz-$version-windows-setup.exe"
```
Comment (Ash258) [inline bucket/veusz.json]:
```suggestion
                "url": "https://github.com/veusz/veusz/releases/download/veusz-$version/veusz-$version-windows-setup.exe#/dl.7z"
```

File: bucket/veusz.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    "description": "a scientific plotting package",
+    "license": "GPL-2.0-or-later",
+    "version": "3.1",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/veusz/veusz/releases/download/veusz-3.1/veusz-3.1-windows-setup.exe#/dl.7z",
+            "hash": "b91ca4de46e71ed61d4e22dd96250a4b7dcb08aef872b91e0bc4c44f6eb62f91"
+        }
+    },
+    "bin": [
```
Comment (Ash258) [inline bucket/veusz.json]:
```suggestion
    "bin": "veusz.exe",
```

File: bucket/veusz.json [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+        "veusz.exe"
+    ],
+    "shortcuts": [
+        [
+            "veusz.exe",
+            "Veusz"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/veusz/veusz"
```
Comment (Ash258) [inline bucket/veusz.json]:
```suggestion
        "github": "https://github.com/veusz/veusz",
        "regex": "tree/veusz-([\\d.]+)"
```

File: bucket/veusz.json [hunk 6]
```diff
@@ -0,0 +1,30 @@
+{
+    "homepage": "https://veusz.github.io",
+    "description": "Scientific plotting and graphing program, designed to produce publication-ready 2D and 3D plots.",
+    "license": "GPL-2.0-or-later",
+    "version": "3.1",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/veusz/veusz/releases/download/veusz-3.1/veusz-3.1-windows-setup.exe#/dl.7z",
```
Comment (Ash258) [inline bucket/veusz.json]: Where is said this file is 64bit only?

Comment (tokiedokie) [reply bucket/veusz.json]: sorry this software is works with 32bit
my computer is 64bit so i thought this works with 64bit

how do i fix it?
should i separate 64bit and 32bit using same url or not ?

Comment (Ash258) [reply bucket/veusz.json]: Just remove architecture property. And keep `url` and `hash` in root of manifest

Comment (tokiedokie) [reply bucket/veusz.json]: updated
dcaeb49

File: bucket/veusz.json [hunk 7]
```diff
@@ -0,0 +1,22 @@
+{
+    "homepage": "https://veusz.github.io",
+    "description": "Scientific plotting and graphing program, designed to produce publication-ready 2D and 3D plots.",
+    "license": "GPL-2.0-or-later",
+    "version": "3.1",
+    "url": "https://github.com/veusz/veusz/releases/download/veusz-3.1/veusz-3.1-windows-setup.exe#/dl.7z",
+    "hash": "b91ca4de46e71ed61d4e22dd96250a4b7dcb08aef872b91e0bc4c44f6eb62f91",
```
Comment (Ash258) [inline bucket/veusz.json]:
```suggestion
    "hash": "b91ca4de46e71ed61d4e22dd96250a4b7dcb08aef872b91e0bc4c44f6eb62f91",
    "pre_install": "Remove-Item \"$dir\\`$PLUGINSDIR\" -Force -Recurse",
```

**Pull request merged**